tajo-core/src/main/java/org/apache/tajo/session/Session.java

```diff
@@ -121,6 +124,10 @@ public synchronized String getCurrentDatabase() {
     return currentDatabase;
   }
 
+  public synchronized void setQueryCache(LoadingCache<String, Expr> cache) { this.cache = cache; }
```

Comment:
I've already read Jira. I concerned about whether the cache is maintained within a session. But it was my misunderstanding. I'm not familiar with LoadingCache. Sorry. 

Anyway, it would be better if this cache initialization is moved to Session.